### ISSUE TITLE
Unconfirmed tx information is not being displayed

### ISSUE THREAD

**Issue by jseagrave21** [neo-ngd/Happynodes#85]
I only see a blank screen whenever I click on an unconfirmed tx. I spoke with @wy who said the remote node should be called to retrieve the tx info.
Below is a selected unconfirmed tx:
![unconfirmed tx](https://user-images.githubusercontent.com/40873301/44596727-33b6b880-a782-11e8-8ede-4af8e013642b.png)

This shows the blank screen that is displayed corresponding to the unconfirmed tx:
![blank unconfirmed tx](https://user-images.githubusercontent.com/40873301/44596726-33b6b880-a782-11e8-9641-0b4766f72009.png)

**Comment by i25959341** [comment]
Thanks, I will investigate on this

**Comment by jseagrave21** [comment]
Hey there it looks like this is working now! @wy prompted me to check after the point release. Good work!

Only other comment was that it was not immediately obvious to me that it was working because I looked on my phone and the tx return is below the unconfirmed list so you have to know to scroll down to it. But that is just my opinion. Keep up the go

**Comment by i25959341** [comment]
Looks there is still some issues, will look into it more

**Comment by wy** [comment]
Can you investigate node 585 @i25959341 , looks like it is working fine with other nodes but 585 isn't returning.

**Comment by wy** [comment]
API response is showing:
```
<html>
    <head>
        <meta http-equiv="content-type" content="text/html;charset=utf-8">
        <title>502 Server Error</title>
    </head>
    <body text=#000000 bgcolor=#ffffff>
        <h1>Error: Server Error</h1>
        <h2>The server encountered a temporary error and could not complete your request.
            <p>Please try again in 30 seconds.
            </h2>
            <h2></h2>
        </body>
    </html>
```

**Comment by wy** [comment]
I think it might be because of this:
requests.exceptions.SSLError: HTTPSConnectionPool(host='54.255.161.226', port=10331): Max retries exceeded with url: / (Caused by SSLError(CertificateError("hostname '54.255.161.226' doesn't match either of 'switcheo.network', '*.switcheo.network'",),))

**Comment by wy** [comment]
This is node 585

requests.exceptions.SSLError: HTTPSConnectionPool(host='52.77.130.101', port=10331): Max retries exceeded with url: / (Caused by SSLError(CertificateError("hostname '52.77.130.101' doesn't match either of 'switcheo.network', '*.switcheo.network'",),))

**Comment by i25959341** [comment]
Fixed #93
